src/tools/pontoon/adding_new_project.md

```diff
@@ -34,8 +34,11 @@ Localization for the <a href="https://testpilot.firefox.com/experiments/min-vid"
 * Priority: select priority level from one of the 5 levels available (Lowest, Low, Normal, High, Highest).
 * Contact: select the L10n driver in charge of the project, probably yourself.
 * External Resources: provide links to external resources like l10n preview environment. You need to enter the name and the URL for each resource. You can also pick one of the predefined names: Development site, Production site, Development build, Production build, Screenshots, Language pack.
+* Visibility: determines if the project is visible to the selected users. Pontoon supports the following visibility types:
```

Comment:
> determines if the project is visible to the selected users

I'd say "determines who can access the project".

Reply:
Thanks, that sounds better!